ReactAndroid/src/main/java/com/facebook/react/devsupport/DevSupportManagerImpl.java

```diff
@@ -119,6 +122,8 @@
   private @Nullable RedBoxDialog mRedBoxDialog;
   private @Nullable AlertDialog mDevOptionsDialog;
   private @Nullable DebugOverlayController mDebugOverlayController;
+  private @Nullable DevLoadingView mDevLoadingView;
+  private boolean mDevLoadingViewVisible = false;
```

Comment:
When backgrounding the app we need to hide this view (since it's a system overlay) so we use this flag to restore it to the proper visibility after.